app/src/main/res/menu/menu_search.xml

```diff
@@ -5,7 +5,7 @@
     <item
         android:id="@+id/action_search"
         android:title="@string/menu_search_button"
-        android:icon="@drawable/ic_search_white_24dp"
+        android:icon="@drawable/ic_search_blue_24dp"
```

Comment:
We want to keep its color for dark theme as white. It should be blue only for light theme. You should be using attrs

Reply:
@neslihanturan ->**in image vector it use android:tint="?attr/menu_item_tint" which solve color bug for dark and light theme see below code**   
### <vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24"
    android:tint="?attr/menu_item_tint"
    android:alpha="0.8">